toxcore/tox_private.h

```diff
@@ -139,6 +139,13 @@ typedef enum Tox_Err_Dht_Get_Nodes {
 bool tox_dht_get_nodes(const Tox *tox, const uint8_t *public_key, const char *ip, uint16_t port,
                        const uint8_t *target_public_key, Tox_Err_Dht_Get_Nodes *error);
 
+/**
+ * This function returns the ratio of close dht nodes that are known to support announce/store.
+ *
+ * @return ratio
+ */
+float tox_dht_get_announce_capability_ratio(const Tox *tox);
```

Comment:
Can we return a uint between 0 and 100? We don't currently have floats in the tox (or toxav) API. I'd prefer not to add them.